File: comms/push-notifications/fetch-push-notification/fetch-push-notification.rb
require "twilio"

client = Twilio::Client.new(
  account_id: "<username>",
  auth_token: "<password>"
)

client.push_notifications.fetch(push_notification_id: "comms_pushnotification_01h9krwprkeee8fzqspvwy6nq8")

File: comms/push-notifications/fetch-push-notification/fetch-push-notification.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "TWILIO_ACCOUNT_SID",
        authToken: "TWILIO_AUTH_TOKEN",
    });
    await client.pushNotifications.fetch("comms_pushnotification_01h9krwprkeee8fzqspvwy6nq8");
}
main();
